examples/kms/vault/csi-kms-connection-details.yaml

```diff
@@ -35,6 +35,17 @@ data:
     {
       "encryptionKMSType": "metadata"
     }
+  user-ns-secrets-metadata-test: |-
+    {
+      "encryptionKMSType": "metadata",
+      "secretName": "usersecret",
```

Comment:
Users will take this as a template/default for their configuration. It might be good to use a different name for the secret that is more descriptive.

Reply:
@nixpanic , Does the following sound better suited? 
```yaml
  user-ns-secrets-metadata-test: |-
    {
      "encryptionKMSType": "metadata",
      "secretName": "storage-encryption-secret",
      "secretNamespace": "default"
    }
  user-secrets-metadata-test: |-
    {
      "encryptionKMSType": "metadata",
      "secretName": "tenant-storage-encryption-secret"
    }
```